docs/en/03_Upgrading/index.md

```diff
@@ -10,9 +10,66 @@ your SilverStripe installation means overwriting files, flushing the cache and u
 See our [upgrade notes and changelogs](/changelogs/4.0.0) for 4.0.0 specific information, bugfixes and API changes.
 </div>
 
-## Composer
+## Understanding and planning your upgrade {#planning}
+
+How easy will it be to update my project? It's a fair question, and sometimes a difficult one to answer.
+
+* SilverStripe follows _semantic versioning_ (see our [release process](/contributing/release_process) for details).
+  * "Major" releases introduces API change that may break your application.
+  * "Minor" releases (x.y) introduces API changes in a backward compatible way and can mark some API as deprecated.
+  * "Patch" releases (x.y.z) fix bugs without introducing any API changes.
+* If you've made custom branches of SilverStripe core, or any thirdparty module, it's going to be harder to upgrade.
+* The more custom features you have, the harder it will be to upgrade. You will have to re-test all of those features, and adapt to API changes in core.
+* Customizations of a well defined type - such as custom page types or custom blog widgets - are going to be easier to upgrade than customisations that modify deep system internals like rewriting SQL queries.
+
+### Overview of changes
```

Comment:
I assume you're still working on consolidating this with the "list of major API changes" further down? There should be one list in the upgrading guide (for highlevel technical decision makers), and then a detailed list in the individual changelog documents.

Reply:
That's been updated with what strikes me as the most important points.